.github/workflows/create-release-branch.yaml

```diff
@@ -40,19 +40,6 @@ jobs:
           sudo chmod +x /usr/local/bin/k
       - name: Build aks-engine binary
         run: make build-binary
-      - name: Validate 1.18 no-egress scenario

```

Comment:
Nice!

While you're here can we add 1.22 for both of our release GitHub Actions?